data/initrd/scripts/remote_log_setup

```diff
@@ -0,0 +1,66 @@
+#! /bin/sh
+#USAGE: ./script_name $remote_log_host_ip
+
+#preconditions
+echo -n "linking mkdir"
+if  ln -s /usr/bin/mkdir /bin/mkdir ; then
+  echo " ok"
+else
+  echo " failed"
```

Comment:
Does it make sense to continue when something fails? I guess it will fail later anyway...